docs/website/docs/index.md

```diff
@@ -23,7 +23,6 @@ IREE supports importing from a variety of ML frameworks:
 - [x] TensorFlow
 - [x] TensorFlow Lite
 - [x] JAX
-- [x] PyTorch
 - [ ] ONNX (hoped for)
```

Comment:
I'd like to talk through this more before dropping PyTorch from this list entirely (that seems like a much larger topic than "quietly remove it from the docs"). As mentioned on https://github.com/iree-org/iree/issues/12034#issuecomment-1423042586, can we start by pointing users at https://github.com/nod-ai/SHARK or somewhere other than iree-torch?

The JAX page can be an example of a "under construction" page: https://iree-org.github.io/iree/getting-started/jax/

We really need to stand behind commitments like framework support though. I'm pretty concerned that these pages and integrations are slipping.

Reply:
I can update to link to SHARK, but I don't know what we'd put on an under construction page for PyTorch.  The JAX page says it's being actively worked on; in this case, however, it isn't.

Reply:
Updated the PR based on our chat offline (used the language you suggested)